\documentclass[11pt]{article}
\usepackage{amsmath}
\usepackage{amssymb}
\usepackage[backend=bibtex, style=authoryear]{biblatex}

% \addbibresource{~/Dropbox/TAGreview/PhDreferences.bib}

\textwidth=470pt
\oddsidemargin=0pt
\topmargin=0pt
\headheight=0pt
\textheight=650pt
\headsep=0pt

\title{Statement of Research}


\author{Nicholas Santantonio}
\date{\today}


\newcommand{\gxe}{G$\times$E}
\newcommand{\gxg}{G$\times$G}


\begin{document}

\section*{\centering Statement of Research}
\begin{center} Nicholas Santantonio \end{center}

% \subsection*{Introduction}

% gxg cows
% cover crop to sequester carbon, 
% htp, 
% weather data
% crop modeling,
% abiotic biotic adaptation
% climate change
% biofuel
% sustainable cropping systems. 


% digital ag
% connect with engineering dept. 
% Fed capacity Hatch funds used only for breeding  

% Lead by Professors Royse Murphy and Don Viands, the Cornell forage breeding program has a rich legacy of providing forage crop varieties to farmers in the Northeast US, as well as collaboration nation-wide, a legacy that I hope to contribute to. 

% Forage research has been drastically under-funded compared to food crops, but renewed interest in sustainable farming has brought them back into the limelight. 

\noindent Small grains comprise an important part of Virginia agriculture, providing flexibility and diversity to the agricultural system. To compete with private small grains seed companies, the public breeding program must implement the latest breeding technology to acheive population improvement alongside varietal development and maximize the genetic response to Virginia environments. 

% By maximizing the genetic response to Virginia mega-environments, and minimizing genetic response to yearly variation, . 

%Processing peppers have also become more important as Americans seek a little more spice in their culinary lives. 

I intend to develop the foundational capacity to implement $21^\text{st}$ century breeding technology in the Small Grains Breeding Program at Virginia Tech, providing ample opportunities for extramural funding. This will include the collection, storage and accessibility of genome-wide information, pedigrees, high-throughput phenotypes collected through proximal sensing, and traditional phenotypes from field trials. Ground-breaking breeding technologies will be implemented to demonstrate the potential of these new resources and pique public interest in crop improvement at Virginia Tech.  

\subsection*{Short-term gains}

% I intend to use wheat and barley as model organisms to ask important questions about how to transition a traditional breeding program into a $21^\text{st}$ century breeding program

The most easily exploited terms in the breeders equation are cycle time and selection intensity. The number of lines evaluated can be drastically increased by reducing or eliminating replication in early-stage trials, thus increasing selection intensity. Genotyping some lines within each family will increase reliability and allow for estimation of genetic correlations of locations and prediction of genetic merit within locations. All materials that are advanced to the second year yield trials will be genotyped. Aerial phenotypes obtained through proximal sensing will be used in multi-trait models to substantially increase trial sizes and reduce the number of plots that are harvested. High-throughput phenotypes will also be used to account for spatial variation within the field for better estimation of genetic values of unreplicated lines, as well as monitor diseases such as scab.% \emph{Fusarium} head blight (scab). 

Initially, the breeding cycle will be shortened by recycling materials after the second year of yield trials, using genotypic information and mathematical optimization to drive crossing decisions. A Smith-Hazel economic selection index including grain yield, days to maturity, deoxynivalenol (DON) content, and test weight will be used as the genetic merit criterion to transition to a optimal contribution genomic selection program. Other properties, including milling and baking quality in wheat, forage and malting quality in barley, straw content and quality, as well as other disease resistances will all be monitored and included in the decision making process. 

\subsection*{Long-term sustainability}

The infrastructure to support regular genotyping and proximal sensing must be developed before the program is overwhelmed by incoming data. While I intend to host a local breeding program database, I would like to work with the T3 database to host proximal sensing images along with phenotypic trial data, pedigrees and genotypes. Quality control of genetic markers, pedigrees and phenotypes must be implemented and standard operating procedures must be developed. Genotyping will allow for construction of a multi-trait, multi-environment genomic prediction model that will be updated yearly. Field experimental design will then be optimized to leverage all phenotypic information available, using genotypic information to link otherwise unrelated trials and traits.

% All phenotypic information will be be connected through pedigrees and genome-wide markers. Predictive models will be updated with new individuals and phenotypes on a yearly basis.

Once genotyping and rapid recycling has been hammered out and sufficient genotypic and phenotypic information is generated ($\leq 5$ years), a recurrent population will be constructed using current breeding materials. This population will be subjected to rapid cycling twice a year to increase the genetic merit of the population, generate unique meiotic recombinations and minimize inbreeding using optimal contribution. Materials will be pulled out of the recurrent population on a yearly basis for two generations of high intensity genomic selection and two generations of inbreeding in the greenhouse. Family sizes will vary based on predicted family means and variances. Materials will also be pulled directly out of the recurrent population and phenotyped for generating information highly connected to the recurrent population to increase accuracy of selection. 


% . Eventually, rapid-cycle genomic selection using mathematical optimization will be implemented for expedited improvement of quantitative traits based on economic selection indices.



% once phenotypic information from that field season is collected. Bayesian models will be used to update the previous years predictive model with new information and individuals from the current year.  
% To accelerate improvement of complex traits, the breeding program must take advantage of the latest breeding technologies. While there has been much discussion on genomics-assisted breeding efforts, very little literature focuses on the actual implementation. 

% While wheat and barley are both self-pollinated annuals with similar haploid chromosome structures, they have very different genome architecture, with the former maintaining three unique genomes and the latter only shelving one. This provides a convenient contrast for evaluating the impact of new breeding technologies on duplicated and non-duplicated genomes. 


% A low cost, moderate density genotyping amplicon based platform will need to be developed for wheat and barley to enable genotyping of all materials represented in recent field trials, as well as all new material that enters into the phenotypic pipeline. 


\subsection*{Breeding for synergistic organism interactions}

The wheat-soybean double cropping system has allowed farmers to profit from growing winter wheat by production of a late season soybean crop, but breeding efforts have largely focused on improving each crop independently. I envision an exciting collaboration between the small grains and soybean breeding programs to breed for synergistic combinations suited for the double cropping system. 

% Important wheat traits would include fall vigor, early maturity and uniform nitrogen use, while important soybean traits include establishment in wheat stubble and early late-season maturity, along with all the other agronomic traits that make these crops viable options for growers. Similarly, an economic index comprising of both the wheat and soybean output could be constructed to maximize total season profitability. 

While the number of potential wheat-soybean pairs is very large and generally intractable, clever testing of select pairs will allow for genomic prediction of unobserved combinations. This is similar to the G$\times$E problem, where one crop (e.g. wheat) can be thought of as an environment to which the genotype (e.g. soybean) is subjected to. In this case, the genetic correlation of genotypes \emph{and} environments would be estimated with genome-wide markers. All pairwise combinations would then be predicted using multivariate linear mixed models. Promising predicted pairs would then be evaluated the following season, and crosses made to maximize the beneficial interaction effect. %While each round of genomic interaction selection would take at least two years to complete depending on the number of trial years desired, the process could improve both crops toward one another. 

% While the genetic covariance of environments is generally unknown and must be estimated \emph{post hoc}, in this case, \emph{a priori} estimates of genetic covariance between genotypes (say for wheat is $\mathbf{A}$) and ``environments'' (say for soybeans is $\mathbf{B}$) can be obtained using genome-wide markers. When both covariances are known, the covariance of all combinations is simply $\mathbf{A} \otimes \mathbf{B}$, 


These synergistic genomic selection models could also be adapted to other goals, allowing for much wider collaborations between animal breeders, microbiologists and soil scientists. Examples include forage/animal and forage/rumen-microbiome using forage barley as a model, root/soil microbiome using wheat as a model, and even barley/yeast interactions using malting barley as a model. Products specifically improved for one another could be marketed as a package (e.g. maltsters could offer specific yeasts with certain barley malts), or a service (e.g. genotyping a farmers field to determine optimal varieties for their soil microbiome). Synergistic improvement of multiple organisms has only just recently become feasible due to cost-effective genotyping platforms, and has the potential to change the way we breed within our food systems. 



\subsection*{Research Philosophy}

In the era of big data, the sheer amount of testable hypotheses is seemingly limitless. A shift away from small designed experiments to large observational studies at the breeding program or whole organism scale is inevitable. A traditional breeding program generates a plethora of phenotypic data that is used to make yearly breeding decisions, and subsequently discarded. If genotyped, these materials become treasure troves of data for asking questions, as well as making breeding decisions. This does not mean that we should cease the design and execution of experiments to address specific hypotheses, but we cannot ignore the valuable resource of observational data being collected, typically at great expense. Genotyping at this scale is feasible given the drastic reduction in costs and availability of third party services, and can be offset by clever experimental design that trades replication at an individual level for replication at a genetic level.

I believe in the collaborative model, where breeding programs do not operate in isolation. They share germplasm, resources, expertise, and most importantly, ideas. Unlike germplasm, ideas also have the merit of being species flexible. I intend to build a collaborative effort at Virginia Tech to aid all breeding programs to build foundational capabilities to increase efficiency of varietal development. As climate change progresses, Virginia will likely experience rising temperatures and more severe weather events, making crop production more difficult. Heat, drought, intense storms and hard frosts will be the new norm, and we must work together to do our part in defending our food security through accelerated genetic improvement. 

\end{document}
